packages/fluent-theme/src/components/Theme.module.css

```
@@ -0,0 +1,60 @@
+
+:global(.webchat-fluent).theme {
+  display: contents;
+
+  --webchat-colorNeutralForeground1: var(--colorNeutralForeground1, #242424);
```

Maybe this one later:

```suggestion
  --webchat--colorNeutralForeground1: var(--colorNeutralForeground1, #242424);
```

Or...

```suggestion
  --webchat-fluent--colorNeutralForeground1: var(--colorNeutralForeground1, #242424);
```

---

Let's address in the next PR